lumen/dashboard.py

```diff
@@ -110,8 +110,7 @@ class Config(Component):
     title = param.String(default="Lumen Dashboard", constant=True, doc="""
         The title of the dashboard.""")
 
-    template = param.Selector(default=_TEMPLATES['material'], constant=True, objects=_TEMPLATES,
-                              check_on_set=False, doc="""
+    template = param.Parameter(default=_TEMPLATES['material'], constant=True, doc="""
```

Comment:
Was trying to use `ClassSelector(class_=(dict, str, pn.template.Template), is_instance=False)` but didn't let me so I used Parameter instead since there's validator methods anyways.

Reply:
Sounds fine, but generally we shouldn't shy away from creating custom Parameter types.

Reply:
Actually, I've never done that. How do I create custom Parameter types?

Reply:
Just subclass Parameter and implement whatever validation logic you need, some examples outside of Param: https://github.com/holoviz/panel/blob/main/panel/_param.py